plugins/bmc_helix_itsm/plugin.spec.yaml

```diff
@@ -0,0 +1,638 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: bmc_helix_itsm
+title: BMC Helix ITSM
+description: BMC Helix IT Service Management is a cloud-native SaaS solution leveraging AI and automation to accelerate IT team results
+version: 1.0.0
+supported_versions: ["BMC Helix ITSM 21.30.00"]
+vendor: rapid7
+support: rapid7
```

Comment:
Should this have cloud_ready enabled ?

Reply:
Following discussion with @wjohnstone-r7 , this plugin will not be cloud enabled.